gohai.go

```diff
@@ -87,8 +99,41 @@ func versionString() string {
 	return buf.String()
 }
 
+// Implement the flag.Value interface
+func (sc *SelectedCollectors) String() string {
+	collectorSlice := make([]string, 0)
+	for collectorName, _ := range *sc {
+		collectorSlice = append(collectorSlice, collectorName)
+	}
+	return fmt.Sprint(collectorSlice)
+}
+
+func (sc *SelectedCollectors) Set(value string) error {
+	for _, collectorName := range strings.Split(value, ",") {
+		(*sc)[collectorName] = true
+	}
+	return nil
+}
+
+// Return whether we should collect on a given collector, depending on the parsed flags
+func shouldCollect(collector Collector) bool {
+	if _, ok := options.only[collector.Name()]; len(options.only) > 0 && !ok {
```

Comment:
Yes exactly, that's also why I named the option `only` and not `include`. A call of `gohai` without options would use all the collectors.